src/_utils/item/Item.tsx

```diff
@@ -47,6 +47,7 @@ export interface ItemProps extends A11yProps {
   readonly onMouseDown?: (event: React.MouseEvent<HTMLElement>) => void
   readonly hideHoverBackground?: boolean
   readonly disabled?: boolean
+  readonly inBorderedComponent?: boolean
```

Comment:
API: _wrapped_? _isWrapped_? _contained_? _hasContainer_? _hasSpacing_? (to have an alusion with `nospacing`)

Reply:
Went with `hasHorizontalSpacing` & `noHorizontalSpacing`